src/main/java/com/revature/domain/EvalComment.java

```diff
@@ -0,0 +1,98 @@
+package com.revature.domain;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+@Table(name="ie_eval_comment")
+public class EvalComment {
+	
+	@Id
+	@Column(name="ec_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="evalCommentSeq")
+	@SequenceGenerator(allocationSize=1, name="evalCommentSeq", sequenceName="eval_comment_seq")
+	private Integer id;
+	
+	@Column(name="ec_comment_text")
+	private String commentText;
+	
+	@ManyToOne(fetch=FetchType.EAGER)
+	@JoinColumn(name="ec_eid")
+	private Eval eval;
+	
+	public EvalComment(){}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)